sql/field.h

```diff
@@ -2194,10 +2194,15 @@ class Field_temporal :public Field {
   uint8 dec; // Number of fractional digits
 
   /**
-    Adjust number of decimal digits from NOT_FIXED_DEC to DATETIME_MAX_DECIMALS
+    Adjust number of decimal digits from NOT_FIXED_DEC to DATETIME_MAX_DECIMALS,
```

Comment:
Why 5.7 does not produce this warning with seemingly unfixed code?

Reply:
They solved it by calling normalize_dec twice.

```
len_arg + ((dec= normalize_dec(dec_arg)) ? normalize_dec(dec_arg) + 1 : 0),
```

And I wanted to avoid that here with this change.